tests/foreman/cli/test_user.py

```diff
@@ -405,9 +407,21 @@ def test_personal_access_token_admin(self):
 
         :CaseImportance: High
         """
+        user = make_user({'admin': '1'})
+        token_name = gen_alphanumeric()
+        result = User.personal_access_token_create({'name': token_name, 'user-id': user['id']})
+        token_value = result[0]['message'].split('\n')[-1]
+        curl_command = (
+            f'curl -k -u {user["login"]}:{token_value} '
+            f'https://{settings.server.hostname}/api/v2/users'
```

Comment:
If you inherit the default_sat fixture, you can use it to not only execute commands, but you can reference it's hostname directly and no longer need to import `command`

Reply:
Done